app/templates/admin/manage_user.html

```diff
@@ -32,9 +32,13 @@
 {% block content %}
     <div class="ui stackable centered grid container">
         <div class="twelve wide column">
-            <a class="ui basic compact button" href="{{ url_for('admin.registered_users') }}">
+            <a class="ui basic compact button" href="{{ url_for('admin.approved_users') }}">
                 <i class="caret left icon"></i>
```

Comment:
can you add a field or a modal at the top that describes the user as "unapproved" on this page as well?